cmake_modules/dependencies/utopic.cmake

```diff
@@ -1 +1 @@
-SET_DEPENDENCIES ( CPACK_DEBIAN_PACKAGE_DEPENDS libboost-regex1.55.0 libicu52 libxslt1.1 libxml2 binutils libldap-2.4-2 openssl zlib1g g++ openssh-client openssh-server expect libarchive13 rsync libapr1 libaprutil1 python libtbb2 )
+SET_DEPENDENCIES ( CPACK_DEBIAN_PACKAGE_DEPENDS g++ openssh-client openssh-server expect rsync libapr1 python )
```

Comment:
Not clear why zip is needed on saucy but not utopic... (and other similar questions)